src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java

```diff
@@ -41,11 +41,11 @@ public void testGetGenesisTx() {
         String blockHash = "abc123";
         Coin genesisTotalSupply = Coin.parseCoin("2.5");
         long time = new Date().getTime();
-        final List<TxInput> inputs = asList(
+        final List<TxInput> inputs = Arrays.asList(
```

Comment:
In that case I personally also prefer to not use the static import to not obscure the information where asList is defined.